README.md

```diff
@@ -84,81 +86,174 @@ Local development uses [Laravel Herd](https://herd.laravel.com/docs/1/getting-st
     php artisan google-fonts:fetch
     ```
-
 11. Tell Herd to serve the application:
-
       ```bash
       herd link
       ```
-
 12. Install [Mailpit](https://github.com/axllent/mailpit) so that you can access transactional email from the platform:
-
     ```bash
     brew install mailpit
     brew services start mailpit
     ```
-
     Then, make sure that your `.env` file contains the following values:
-
     ```dotenv
     MAIL_MAILER=smtp
     MAIL_HOST=127.0.0.1
     MAIL_PORT=1025
     ```
-
     You will now be able to access mail that the platform sends by visiting http://127.0.0.1:8025 or http://localhost:8025. For more information and additional configuration options, [read the Mailpit documentation](https://github.com/axllent/mailpit).
 
 For comprehensive instructions, consult the [Laravel documentation](https://laravel.com/docs/10.x). Here's an overview
-of how some key tasks can be carried out using Valet:
+of how some key tasks can be carried out using Herd:
+- [Composer](https://getcomposer.org) commands may be executed by using `composer <command>`.  
+- [NVM](https://github.com/nvm-sh/nvm) commands may be executed by using `nvm <command>`.  
+- [NPM](https://docs.npmjs.com/cli) commands may be executed by using `npm <command>`.  
+- [Artisan](https://laravel.com/docs/10.x/artisan) commands may be executed by using `php artisan <command>`.  
+
+### Local development setup using docker compose:
+1. Install docker according to your platform instructions found [here](https://docs.docker.com/get-docker/).
+2. Clone the repository:
+
+    ```bash
+    git clone https://github.com/accessibility-exchange/platform.git && cd platform
+    ```
+
+3. Create a `.env` file from the included example file:
+
+    ```bash
+    cp .env.local.example .env
+    ```
+   
+    Then, change the `APP_ENV` value to `local`:
+
+    ```dotenv
+    APP_ENV=local
+    ```
 
-- [Composer](https://getcomposer.org) commands may be executed by using `composer <command>`.
-- [NVM](https://github.com/nvm-sh/nvm) commands may be executed by using `nvm <command>`.
-- [NPM](https://docs.npmjs.com/cli) commands may be executed by using `npm <command>`.
-- [Artisan](https://laravel.com/docs/10.x/artisan) commands may be executed by using `php artisan <command>`.
+4. Generate an encryption key for [CipherSweet](https://github.com/spatie/laravel-ciphersweet):
+
+    ```bash
+    docker run --rm -it alpine apk add openssl && openssl rand -hex 32
+    ```
+
+    Add it to your `.env` file:
+    
+    ```dotenv
+    CIPHERSWEET_KEY="<your key>"
+    ```
+
+5. Generate your database password:
+
+    ```bash
+    docker run --rm -it alpine apk add openssl && openssl rand -hex 32
+    ```
+    
+    Add it to your `.env` file:
+    
+    ```dotenv
+    DB_PASSWORD="<your key>"
+    ```
+
+6.  Generate an application key:
+
+    ```bash
+    docker compose -f docker-compose.local.yml run --rm --entrypoint '' platform.test php artisan key:generate --show
+    ```
```

Comment:
@marvinroman, this step failed with `dependency failed to start: container platform.redis exited (1)` From the Docker logs related to Redis:

```log
2023-11-02 07:54:48 redis 11:54:48.67 
2023-11-02 07:54:48 redis 11:54:48.67 Welcome to the Bitnami redis container
2023-11-02 07:54:48 redis 11:54:48.68 Subscribe to project updates by watching https://github.com/bitnami/containers
2023-11-02 07:54:48 redis 11:54:48.68 Submit issues and feature requests at https://github.com/bitnami/containers/issues
2023-11-02 07:54:48 redis 11:54:48.69 
2023-11-02 07:54:48 redis 11:54:48.69 INFO  ==> ** Starting Redis setup **
2023-11-02 07:54:48 redis 11:54:48.72 ERROR ==> The REDIS_PASSWORD environment variable is empty or not set. Set the environment variable ALLOW_EMPTY_PASSWORD=yes to allow the container to be started with blank passwords. This is recommended only for development.
```

I didn't see the App key generated either, but I suppose it may be because it didn't get that far.

Reply:
Added creating redis password to the instructions.